website/docs/diff.md

```diff
@@ -9,15 +9,16 @@ sidebar_position: 6
 1. FerretDB uses the same protocol error names and codes, but the exact error messages may be different in some cases.
 2. FerretDB does not support NUL (`\0`) characters in strings.
 3. Document keys must not contain `$` sign.
-4. Database and collection names restrictions:
+4. Replacement documents must not contain `.` sign.
```

Comment:
Any data documents must not contain `.`, the documents we insert mustn't contain `.` either ;)
And I think in this doc all the documents mean "data documents", not commands.